agent/stats/utils_windows_test.go

```diff
@@ -42,39 +42,23 @@ func TestDockerStatsToContainerStatsZeroCoresGeneratesError(t *testing.T) {
 	assert.Error(t, err, "expected error converting container stats with zero cpu cores")
 }
 
-func TestDockerStatsToContainerStatsCpuUsage(t *testing.T) {
-	// doing this with json makes me sad, but is the easiest way to deal with
-	// the inner structs
-
-	// numCores is a global variable in package agent/stats
-	// which denotes the number of cpu cores
-	numCores = 4
-	jsonStat := fmt.Sprintf(`
-		{
-			"cpu_stats":{
-				"cpu_usage":{
-					"total_usage":%d
-				}
-			}
-		}`, 100)
-	dockerStat := &types.StatsJSON{}
-	json.Unmarshal([]byte(jsonStat), dockerStat)
-	containerStats, err := dockerStatsToContainerStats(dockerStat)
-	assert.NoError(t, err, "converting container stats failed")
-	require.NotNil(t, containerStats, "containerStats should not be nil")
-	assert.Equal(t, uint64(2500), containerStats.cpuUsage, "unexpected value for cpuUsage", containerStats.cpuUsage)
-}
-
 func TestDockerStatsToContainerStats(t *testing.T) {
+	numCores = 4
 	inputJsonFile, _ := filepath.Abs("./windows_test_stats.json")
 	jsonBytes, _ := ioutil.ReadFile(inputJsonFile)
 	dockerStat := &types.StatsJSON{}
 	json.Unmarshal([]byte(jsonBytes), dockerStat)
 	containerStats, err := dockerStatsToContainerStats(dockerStat)
 	assert.NoError(t, err, "converting container stats failed")
-	require.NotNil(t, containerStats, "containerStats should not be nil")
```

Comment:
i think this should be require rather than assert. if containerStats is nil, there's no point to check what's inside it

Reply:
I'll make that change.